include/cooperative_perception/ctrv_model.hpp

```diff
@@ -314,6 +314,8 @@ auto nextState(const CtrvState& state, units::time::second_t time_step) -> CtrvS
  */
 auto nextState(const CtrvState& state, units::time::second_t time_step, const CtrvProcessNoise& noise) -> CtrvState;
 
+void debugPrint(const CtrvState& state);
```

Comment:
This should be renamed to something more general. Perhaps `printState`. Also, the syntax should be updated to be consistent with the other functions. It should be:
```cpp
auto printState(const CtrvState& state) -> void;
```